sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/GenerateSchemaTest.kt

```diff
@@ -24,6 +24,34 @@ class GenerateSchemaTest {
     schemaFile.delete()
   }
 
+  @Test fun `generateSchema task can run twice`() {
+    val fixtureRoot = File("src/test/schema-file")
+    val schemaFile = File(fixtureRoot, "src/main/sqldelight/databases/1.db")
+    if (schemaFile.exists()) schemaFile.delete()
+
+    GradleRunner.create()
+        .withProjectDir(fixtureRoot)
+        .withPluginClasspath()
+        .withArguments("clean", "generateMainDatabaseSchema", "--stacktrace")
+        .build()
+
+    // verify
+    assertThat(schemaFile.exists())
+        .isTrue()
+
+    GradleRunner.create()
+        .withProjectDir(fixtureRoot)
+        .withPluginClasspath()
+        .withArguments("clean", "--rerun-tasks", "generateMainDatabaseSchema", "--stacktrace")
+        .build()
+
+    // verify
+    assertThat(schemaFile.exists())
```

Comment:
Maybe check the creation time is different than the first time. Otherwise the task could have no-op'd rather than re-generate.